api/app/controllers/spree/api/v2/platform/menu_items_controller.rb

```diff
@@ -19,7 +19,7 @@ def reposition
             end
 
             if @moved_item.save
-              head :no_content
+              render_serialized_payload { serialize_resource(resource) }
```

Comment:
```suggestion
              render_serialized_payload(204) { serialize_resource(resource) }
```

Reply:
204 response is a no content response, but we are returning the serialised resource.

Reply:
@MatthewKennedy you're right - my bad!